cloud/terraform/aws/backup/modules/backup-plan/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
```

Comment:
Removed the `provider` block so that this could be a reusable module. Also put it in a modules directory. This way the user can instantiate multiple backup plans or vaults.

I believe we should do the same for Liferay, as we have them now, they're root modules, and not reusable modules.